src/Fantomas.Tests/AttributeTests.fs

```diff
@@ -16,7 +16,8 @@ type Funcs =
     |> should equal """[<Extension>]
 type Funcs =
     [<Extension>]
-    static member ToFunc(f: Action<_, _, _>) = Func<_, _, _, _>(fun a b c -> f.Invoke(a, b, c))
+    static member ToFunc(f: Action<_, _, _>) =
```

Comment:
Please don't change the original test, try and tinker with the settings until the existing behaviour is met.
This might not always be possible but the initial attempt should be to leave existing tests as they are.

Reply:
Just to be sure: You do not want any of the existing tests to change? 
The reason I had to change a lot of tests is because the `member`s now respect `MaxBindingWidth` which added line breaks in some of the tests. 
The default value for `MaxBindingWidth` is 40, as was before with `MaxLetBindingWidth`. If the tests are not supposed to change I can either
- change the default value of `MaxBindingWidth` to something higher, like `80`, or
- use a custom `config` for all tests which would have changed with an increased `MaxBindingWidth`

Which of the two options would you prefer?

Reply:
So I prefer you change the config for each test individually to match the previous result.

Reply:
Done.